src/IsolatedTestHost/IsolatedTestHost.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFrameworks>net45</TargetFrameworks>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
```

Comment:
Just so the compiler checks syntax and references in any xml doc comments we define. The warning suppression I also set ensures we aren't obligated to actually write docs.